src/relion/__init__.py

```diff
@@ -44,6 +49,10 @@ def __init__(self, path):
             # )
         self.res = RelionResults()
 
+    @property
+    def plock(self):
+        return PipelineLock(self.basepath)
```

Comment:
Why not give the location of the lockfile as argument?

Does this need to be public?